core/src/main/java/org/apache/druid/common/guava/GuavaUtils.java

```diff
@@ -23,8 +23,8 @@
 import com.google.common.base.Strings;
 import com.google.common.primitives.Longs;
 import org.apache.druid.java.util.common.logger.Logger;
-
 import javax.annotation.Nullable;
+import java.util.ArrayList;
```

Comment:
```[ERROR] /home/travis/build/apache/druid/core/src/main/java/org/apache/druid/common/guava/GuavaUtils.java:26: 'javax.annotation.Nullable' should be separated from previous imports. [ImportOrder]```

The checkstyle wants an empty line between the Lines 26 and 27.

Reply:
ok, it's done.